tetris.py

```diff
@@ -282,6 +293,7 @@ def __init__(self, main_board, small_board, topbar_time):
         self.topbar_time = topbar_time
         self.fast_down = False
         self.start_time = time.time()
+        self.time = 0
```

Comment:
Same data in two places. Please no. Make a function that calculates and returns the time.